lib/MusicLibraryController.rb

```diff
@@ -0,0 +1,95 @@
+require 'pry'
+
+extend Concerns::Findable
+
+class MusicLibraryController
+
+  attr_accessor :path
+
+  def initialize(path = "./db/mp3s")
+    new_importer_obj = MusicImporter.new(path)
+    new_importer_obj.import
+  end
+
+
+  def call
+    input = ""
+    while input != "exit"
+      puts "Welcome to your music library!"
+      puts "To list all of your songs, enter 'list songs'."
+      puts "To list all of the artists in your library, enter 'list artists'."
+      puts "To list all of the genres in your library, enter 'list genres'."
+      puts "To list all of the songs by a particular artist, enter 'list artist'."
+      puts "To list all of the songs of a particular genre, enter 'list genre'."
+      puts "To play a song, enter 'play song'."
+      puts "To quit, type 'exit'."
+      puts "What would you like to do?"
```

Comment:
`call` is a really long method. Maybe break out all of these puts into another method?

Reply:
Shortened my `call` method by creating a separate method called `jawn` & storing all of the strings it wanted to puts out the call method. The result is a much shorter & concise call method.